frontend/model/notifications/templates.js

```diff
@@ -40,7 +44,8 @@ export default ({
       errName: error.name,
       activity,
       action: action ?? opType,
-      contract: state.contracts[contractID]?.type ?? contractID
+      contract: state.contracts[contractID]?.type ?? contractID,
+      errMsg: error.message
```

Comment:
I saw this was added in a recent commit, but I don't see it used. Did you intend to leave it unused?

Reply:
`errMsg` is indeed used, so this is a bugfix.